<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputStrDialog</class>
 <widget class="QWidget" name="OutputStrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output Streams</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="Label10">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Output Stream</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLabel" name="Label5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="Stream1C">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>(4) Solution 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="4">
    <widget class="QComboBox" name="Stream1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="BtnStr1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="3">
    <widget class="QComboBox" name="Format1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Lat/Lon/Height</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X/Y/Z-ECEF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E/N/U-Baseline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NMEA0183</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solution Status</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="Stream2C">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>(5) Solution 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="4">
    <widget class="QComboBox" name="Stream2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="BtnStr2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="3">
    <widget class="QComboBox" name="Format2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Lat/Lon/Height</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X/Y/Z-ECEF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E/N/U-Baseline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NMEA0183</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solution Status</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="LabelF1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Output File Paths</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="10">
    <widget class="QLineEdit" name="FilePath1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QPushButton" name="BtnFile1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="10">
    <widget class="QLineEdit" name="FilePath2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="10">
    <widget class="QPushButton" name="BtnFile2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="TimeTagC">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Time-Tag</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="Label1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Swap Intv</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QComboBox" name="SwapIntv">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="Label2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="BtnKey">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="3">
    <widget class="QPushButton" name="BtnOk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="9" colspan="2">
    <widget class="QPushButton" name="BtnCancel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="Label6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="Label7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
